app/src/main/java/com/fstyle/dialogmanager/MainActivity.java

```diff
@@ -1,13 +1,176 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        mDialogManager = new DialogManagerImpl(this);
+    }
+
+    public void onClick(View view) {
+        switch (view.getId()) {
+            case R.id.btnProgress:
+                showProgress();
+                break;
+            case R.id.btnBasicNoTitle:
+                showDialogBasicWithoutTitle();
+                break;
+            case R.id.btnBasic:
+                showDialogBasic();
+                break;
+            case R.id.btnBasicWithIcon:
+                showDialogBasicWithIcon();
+                break;
+            case R.id.btnBasicCheckPrompt:
+                showDialogBasicCheckPrompt();
+                break;
+            case R.id.btnNeutral:
+                showDialogNeutral();
+                break;
+            case R.id.btnList:
+                showDialogList();
+                break;
+            case R.id.btnListNoTitle:
+                showDialogListWithoutTitle();
+                break;
+            case R.id.btnListSingleChoice:
+                showDialogListSingleChoice();
+                break;
+            case R.id.btnListMultiChoice:
+                showListMultiChoice();
+                break;
+            default:
+                break;
+        }
+    }
+
+    private void showListMultiChoice() {
+        mDialogManager.dialogListMultiChoice("abc", R.array.socialNetworks, new Integer[] { 0, 1 },
+                new MaterialDialog.ListCallbackMultiChoice() {
+                    @Override
+                    public boolean onSelection(MaterialDialog materialDialog, Integer[] integers,
+                            CharSequence[] charSequences) {
+                        for (CharSequence charSequence : charSequences) {
+                            Log.d(TAG, "onSelection: " + charSequence);
+                        }
+                        return true;
+                    }
+                });
+    }
+
+    private void showDialogListSingleChoice() {
+        mDialogManager.dialogListSingleChoice("title", R.array.socialNetworks, 0,
+                new MaterialDialog.ListCallbackSingleChoice() {
+
+                    @Override
+                    public boolean onSelection(MaterialDialog dialog, View itemView, int which,
+                            CharSequence text) {
+                        Log.d(TAG, "onSelection: " + text);
+                        return true;
+                    }
+                });
+    }
+
+    private void showDialogListWithoutTitle() {
+        mDialogManager.dialogBasicWithoutTitle("test abc",
+                new MaterialDialog.SingleButtonCallback() {
+                    @Override
+                    public void onClick(@NonNull MaterialDialog materialDialog,
+                            @NonNull DialogAction dialogAction) {
+                        Log.d(TAG, "onClick: ");
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "onClick: " appears 6 times in this file; the first occurrence is on line 94
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals